sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -5,10 +5,113 @@
 ```ts
 
 import { AbortSignalLike } from '@azure/abort-controller';
+import { BaseRequestPolicy } from '@azure/core-http';
+import { HttpMethods } from '@azure/core-http';
+import { HttpOperationResponse } from '@azure/core-http';
+import { OperationArguments } from '@azure/core-http';
+import { OperationSpec } from '@azure/core-http';
+import { RequestPolicy } from '@azure/core-http';
+import { RequestPolicyOptions } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { WebResource } from '@azure/core-http';
```

Comment:
I am concerned about having `core-lro` depend on `core-http`, since it doesn't seem like it did before. This will make the v2 transition more difficult, since many of these types are renamed, so we'll have to either major version bump `core-lro` or create yet another package, which feels bad.

I was thinking of `core-lro` morally similar to `core-paging` where it provided just a little bit of goodness for helping data plane authors to make the right public shapes, rather than internals for the code generator.

Reply:
I agree with Jeff! Let's not do this.